app/controllers/ui_job_wizard_controller.rb

```diff
@@ -24,7 +25,11 @@ def resource_name(nested_resource = nil)
   end
 
   def map_template_inputs(template_inputs_with_foreign)
-    template_inputs_with_foreign.map { |input| input.attributes.merge({:resource_type => input.resource_type&.tableize }) }
+    template_inputs_with_foreign.map { |input| input.attributes.merge({:resource_type => input.resource_type&.tableize, :url => search_path(input.resource_type&.tableize) }) }
+  end
+
+  def with_katello
+    defined?(::Katello) ? true : false
```

Comment:
`!!defined?(::Katello)` would be more idiomatic. Also method names usually end with a question mark if the return type is a boolean